src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithArrayFWTest.java

```diff
@@ -280,7 +280,8 @@ private static StringFW asStringFW(
         }
     }
 
-    private static List<StringFW> asStringFWArray(StringFW... values)
+    private static List<StringFW> asStringFWArray(
+        StringFW... values)
```

Comment:
Is it sufficient to us `Arrays.asList(asStringFW(...), asStringFW(...), ...)` at the call site instead of having this extra method?

If the compiler is insisting on it, then we can keep the method, but suggest returning `Arrays.asList(values)` directly instead of `new ArrayList<>(Arrays.asList(values))` unless we really need a mutable array that is a copy of the values.